systemdev/testing-locally.rst

```diff
@@ -127,8 +127,8 @@ In both cases the chroot will be created by running the following command::
 
 A chroot for cross-building arm64 packages on an amd64 host can e.g. be created under the directory ``/srv/chroot/ubports-${distro}-arm64`` using::
```

Comment:
The var name needs to be fixed too for consistency

Reply:
Ahh yes, I missed that , I will change it tonight and update the PR, thank you for noticing.